src/js/actions/event/data.js

```diff
@@ -0,0 +1,47 @@
+export const GET_EVENT = 'GET_EVENT';
+export const GET_EVENT_REQUEST = 'GET_EVENT_REQUEST';
+export const GET_EVENT_SUCCESS = 'GET_EVENT_SUCCESS';
+export const GET_EVENT_FAILURE = 'GET_EVENT_FAILURE';
+
+const event1 = {
+  event_id: 1,
+  host_user_id: 1,
+  name: 'Lounge party',
+  description: 'Celebrating life',
+  is_poll: true,
+  is_edited: false,
+  _invitees: [2],
+  _what: ['dancing', 'skydiving'],
+  _where: ['Forest', 'Camping'],
+  _when: ['2017-01-03T00:00:00.000Z', '2017-02-14T00:00:00.000Z']
```

Comment:
@heron2014 Interesting.  So the inputs give us actual timestamps?

So if you select a date and no time, you still get a long timestamp with the current time?  
One of the features in Spark is that a host doesn't have to choose the time of the event straight away.  You **must** choose at least one date, but the time can be left blank until later.  

Combining them should still be possible.  If you convert the timestamps to milliseconds you can just add them together.  I reckon this is something we'll have to look into properly though.

What do you think we should do for the meantime?